docs/team/wuyuxin.md

```diff
@@ -0,0 +1,48 @@
+# Wu Yuxin - Project Portfolio Page
+
+## Overview
+#### Project: Nav@NUS
+Nav@NUS is a desktop application that aids users in getting around NUS. The user interacts with it using a CLI and thus,
+this application is appropriate for users who can type fast.
+
+Given below are my contributions to the project.
+
+* **New Feature**: Added an /addfav command
+  * What it does: allows the user to add a previous command that is valid to their list of favourites.
+  * Justification: This feature improves the product significantly because some users might not be familiar enough with
+   the commands and have to constantly refer to the user guide.
+   Through this command it allows them to save previous commands will greatly improve convenience.
+  * Highlights: This enhancement affects existing commands and commands to be added in the future. 
+  The implementation too was challenging as it required changes to existing commands to determine if command about to be added is valid.
+  
+* **New Feature**: Added a /liststops command.
+ * What it does: allows the user to view all bus stops and their description.
+ * Justification: This feature enhances the usability of this application as users of this application is most likely unfamiliar with all the bus stops in NUS.
+ By listing all bus stops and their description, it allows the user to know more information about the location of the bus stop, and the surrounding infrastructures.
```

Comment:
Indentation looks a little off. might wanna use "+" instead of "*" for easier indentation